include/defines.h

```diff
@@ -21,6 +21,12 @@
 // Compiles to -0x1000 in diff.py
 #define PLAYER_HUMAN_AND_CPU PLAYER_EXISTS | PLAYER_HUMAN | PLAYER_CPU | PLAYER_START_SEQUENCE
 
+// 0 players, 1 player, 2 players, 3 players, or 4 players
+#define NUM_PLAYER_NUM_SELECTIONS 5
```

Comment:
NUM used twice here intentional?

Reply:
Yeahhh I was thrilled with this name. It's meant to be the number of possible 'num players'

Reply:
`MAX_NUM_PLAYERS`?
`NUM_PLAYERS_MAX`?
`FIVE_NUM_PLAYERS_SELECTED`?

Because this is a define remember that it's naming should be based upon the `5` I think. Since it's not a variable. Generally, we should understand what the define is without necessarily going to its definition.

Reply:
I think most people would assume `MAX_NUM_PLAYERS` and `NUM_PLAYERS_MAX` to be 4.

Maybe:
#define ZERO_PLAYERS_SELECTED 0
#define ONE_PLAYERS_SELECTED 1
#define TWO_PLAYERS_SELECTED 2
#define THREE_PLAYERS_SELECTED 3
#define FOUR_PLAYERS_SELECTED 4
#define NUM_PLAYER_SELECTIONS 5 ?

Reply:
TOTAL_SELELECTED_PLAYER_DEFINES ?

Reply:
Works for me